apps/web-giddh/src/app/shared/header/components/new-manage-groups-accounts/manage-groups-accounts.component.html

```diff
@@ -26,53 +21,122 @@ <h3 class="modal-title bg pdT">Manage Groups &amp; Accounts</h3>
+                        </account-operations>
+                    </div>
+
+
                 </div>
-                <div class="search-loading no-result" *ngIf="!(searchLoad | async) && !(groupList$ | async)?.length">
+            </tab>
+            <!--           new -->
+            <tab heading="Custom">
+                <div class="custom-filed-list-wrapper" *ngIf="!isGetCustomInProgress">
+                    <ng-container>
+                        <form [formGroup]="customFieldForm">
+                            <ng-container formArrayName="customField">
+                                <ng-container *ngFor="let item of customFieldForm.get('customField')['controls']; let index=index;let first = first; let last = last">
+                                    <div class="row mb-15" [formGroupName]="index">
+                                        <div class="col-lg-3 col-sm-3 col-xs-12">
+                                            <label *ngIf="first || isMobileScreen">Name</label>
+                                            <input type="text" class="form-control" placeholder="Enter custom field name" formControlName="key" name="key" autocomplete="off" [attr.disabled]="item.get('uniqueName').value && isEnabledIndex!==index ?'': null">
+                                        </div>
+                                        <div class="col-lg-3 col-sm-3 col-xs-12">
+                                            <label *ngIf="first || isMobileScreen">Type</label>
+                                            <sh-select [options]="dataTypeList" formControlName="dataType" name="dataType" (selected)="customFieldTypeSelected($event, index)" [disabled]="item.get('uniqueName').value && isEnabledIndex!==index">
+                                            </sh-select>
+                                        </div>
+                                        <div class="col-lg-3 col-sm-3 col-xs-12">
+                                            <label *ngIf="first || isMobileScreen">Length</label>
+                                            <input type="number" class="form-control" formControlName="valueLength" name="valueLength" [attr.disabled]="(item.get('uniqueName').value &&  isEnabledIndex!==index) || item.get('dataType').value ==='BOOLEAN' ?'': null" autocomplete="off">
+                                        </div>
+                                        <div class="col-lg-3 col-sm-3 col-xs-12 edit-close-btn">
+                                            <label *ngIf="first || isMobileScreen" class="d-block">&nbsp;</label>
+                                            <button class="btn btn-defualt close-button" (click)="openModal(template, index );">
+                                                <i class="icon-trash"></i>
+                                            </button>
+                                            <button *ngIf="item.get('uniqueName').value" class="btn btn-defualt" (click)="editCustomfield(index)">
```

Comment:
Incorrect spelling of default
```suggestion
                                            <button *ngIf="item.get('uniqueName').value" class="btn btn-default" (click)="editCustomfield(index)">
```